tests/operators/test_read.py

```diff
@@ -526,3 +526,46 @@ def test_lfric_forecast_period_standard_name_callback(cube):
     cube.coord("forecast_period").standard_name = None
     read._lfric_forecast_period_standard_name_callback(cube)
     assert cube.coord("forecast_period").standard_name == "forecast_period"
+
+
+def test_read_cubes_extract_subarea():
+    """Read cube and ensure appropriate subarea is extracted."""
+    # All cubes are on same grid, and vary by cell method.
+    cube = read.read_cubes("tests/test_data/air_temp.nc")[0]
+    assert (
+        repr(cube.coord("grid_latitude").coord_system)
+        == "RotatedGeogCS(37.5, 177.5, ellipsoid=GeogCS(6371229.0))"
```

Comment:
```suggestion
        cube.coord("grid_latitude").coord_system
        == iris.coords.RotatedGeogCS(37.5, 177.5, ellipsoid=GeogCS(6371229.0))"
```
Its better to compare the actual object if you can.